addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml

```diff
@@ -0,0 +1,109 @@
+
+	<thing-type id="mill">
+		<label>WindCentrale Wind Mill</label>
+
+		<channels>
+			<channel id="windSpeed" typeId="windSpeedType" />
+			<channel id="windDirection" typeId="windDirectionType" />
+			<channel id="powerAbsTot" typeId="powerKType">
+				<label>Total Power</label>
+			</channel>
+			<channel id="powerAbsWd" typeId="powerType">
+				<label>WD Power</label>
+			</channel>
+			<channel id="powerRel" typeId="powerRelType">
+			</channel>
+
+			<channel id="kwh" typeId="energyType">
+			</channel>
+			<channel id="kwhForecast" typeId="energyType">
+				<label>Energy Forecast</label>
+			</channel>
+			<channel id="runPercentage" typeId="powerRelType">
+				<label>Run Percentage</label>
+				<description>Run Percentage this year</description>
+			</channel>
+			<channel id="timestamp" typeId="timestampType" />
+		</channels>
+
+		<properties>
+			<property name="vendor">WindCentrale</property>
+		</properties>
+
+		<config-description>
+			<parameter name="millId" type="integer" required="true">
+				<label>WindMill</label>
+				<options>
+					<option value="1">De Grote Geert</option>
+					<option value="2">De Jonge Held</option>
+					<option value="31">Het Rode Hert</option>
+					<option value="41">De Ranke Zwaan</option>
+					<option value="51">De Witte Juffer</option>
+					<option value="111">De Bonte Hen</option>
+					<option value="121">De Trouwe Wachter</option>
+					<option value="131">De Blauwe Reiger</option>
+					<option value="141">De Vier Winden</option>
+					<option value="191">De Boerenzwaluw</option>
+				</options>
+				<default>131</default>
+			</parameter>
+			<parameter name="wd" type="integer" required="false">
+				<label>Wind Parts</label>
+				<description># of wind parts ("winddelen")</description>
+				<default>1</default>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" min="1" max="9999" required="false">
+				<label>Refresh Interval</label>
+				<description>Refresh Interval for refreshing the data in s</description>
+				<default>30</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="windSpeedType">
+		<item-type>Number</item-type>
+		<label>Wind Speed</label>
+		<category>Wind</category>
+		<state pattern="%.1d Bft" readOnly="true" />
+	</channel-type>
+	<channel-type id="windDirectionType">
+		<item-type>String</item-type>
+		<label>Wind Direction</label>
+		<state pattern="%s" readOnly="true" />
+	</channel-type>
+	<channel-type id="powerRelType">
+		<item-type>Number</item-type>
+		<label>Relative Power</label>
+		<category>Energy</category>
+		<state pattern="%.1f %%" readOnly="true" />
+	</channel-type>
+	<channel-type id="powerType">
+		<item-type>Number</item-type>
+		<label>Power</label>
+		<category>Energy</category>
+		<state pattern="%.1f Wh" readOnly="true" />
```

Comment:
Power (Watt) and Energy (Watt/hour) are two different things. If the consumed power of a LED light is 10W then after 1 hour it has consumed 10Wh of energy (or. 0.01kWh). The same goes for the windmill instead it produces instead of consumes. So you when there is "Power" in the label I would expect a "W" or "kW" unit. The category might also be wrong. But ESH is not clear about this.